src/patternfly/components/Switch/switch.hbs

```diff
@@ -0,0 +1,14 @@
+<label for="{{id}}" class="pf-c-switch{{#if disabled}} pf-m-disabled{{/if}}" role="switch" aria-label="..."  aria-checked="{{#if off}}false{{else}}true{{/if}}">
```

Comment:
We can't keep the `role="switch"` on the `<label>` because it will break the experience in JAWS due to checkbox [inception](https://en.wikipedia.org/wiki/Inception). Let's move it to the `<input>` instead (i.e. same basic functionality as a checkbox, but extending it's role to be the switch).

The [Material switch component](https://material-components.github.io/material-components-web-catalog/#/component/switch) includes `role="switch"` on the `<input>` which worked really well when I tested it.